pkg/cloud/awsprovider.go

```diff
@@ -1290,8 +1290,8 @@ func (a *AWS) ExternalAllocations(start string, end string, aggregators []string
 			%s,
 			SUM(line_item_blended_cost) as blended_cost
 		FROM %s as cost_data
-		WHERE (%s='%s') AND line_item_usage_start_date BETWEEN date '%s' AND date '%s'
-		GROUP BY %s`, aggregatorNames, filter_column_name, customPricing.AthenaTable, filter_column_name, filterValue, start, end, groupby)
+		WHERE (%s='%s') AND line_item_usage_start_date BETWEEN date '%s' AND date '%s AND %s <> '' 
```

Comment:
Are the quotes correct here? Seems like it should be `... AND date '%s' AND...` (with a close quote on the %s) but perhaps I'm just reading this wrong.

Reply:
And why only use `formattedAggregators[0]`?

Reply:
Fixed, thanks. It turns out I was testing the wrong if clause...

Reply:
Ah that makes sense. This one looks better :+1: